lib/unified_search_builder.rb

```diff
@@ -397,4 +398,14 @@ def devolved_org_boost
     }
   end
 
+  def entities_boost
+    {
+      filter: { terms: { entities: find_entities} },
+      boost: 20
+    } unless find_entities.empty?
+  end
+
+  def find_entities
```

Comment:
This could just as easily be `entities` instead of `find_entities`, especially since it's memoized.